manifests/install.pp

```diff
@@ -12,10 +12,13 @@
     ensure => $foreman_proxy::version,
   }
 
-  if $foreman_proxy::log == 'JOURNALD' {
+  if $foreman_proxy::log == 'JOURNAL' {
     package { 'foreman-proxy-journald':
       ensure => installed,
     }
+    if $foreman_proxy::repo {
+      Foreman::Repos['foreman_proxy'] -> Package['foreman-proxy-journald']
+    }
```

Comment:
This could also be

```puppet
Foreman::Repos<| title == 'foreman_proxy' |> -> Package['foreman-proxy-journald']
```

and thus avoid the `if`. It could also be `Foreman::Repos <| |> -> …`

Whether this is more readable or not, I don't know :)

Reply:
I also considered introducing a tag to collect on (https://github.com/theforeman/puppet-foreman/blob/a14059fdcb7701336396a570def254588e7f955e/manifests/compute/ovirt.pp#L12) but since this may need a cherry pick I decided to keep this PR simple.